runtime/src/block_min_prioritization_fee_cache.rs

```diff
@@ -0,0 +1,595 @@
+use {
+    crate::{
+        block_min_prioritization_fee::*,
+        block_min_prioritization_fee_cache_query::BlockMinPrioritizationFeeCacheQuery,
+        block_min_prioritization_fee_cache_update::BlockMinPrioritizationFeeCacheUpdate,
+    },
+    solana_sdk::{
+        clock::Slot, pubkey::Pubkey, saturating_add_assign, transaction::SanitizedTransaction,
+    },
+    std::collections::HashMap,
+};
+
+/// The maximum number of blocks to keep in `BlockMinPrioritizationFeeCache`; States from
+/// up to 150 recent blocks should be sufficient to estimate minimal prioritization fee to
+/// land transactions to current block.
```

Comment:
```suggestion
/// The maximum number of blocks to keep in `BlockMinPrioritizationFeeCache`
```

Reply:
This makes me realize: the way this is currently coded, the cache will keep and report data from any processed/frozen Bank, not just those that end up confirmed. Are there any issues with this?

Reply:
Yeah I think so too. Frozen bank even not being confirmed eventually still carries useful fee information,